server/container/spring/src/main/java/org/apache/james/container/spring/tool/James23ImporterManagement.java

```diff
@@ -69,23 +56,8 @@ public void importMailsFromJames23(String james23MailRepositoryPath) throws Exce
         try {
             james23Importer.importMailsFromJames23(james23MailRepositoryPath);
         } catch (Exception e) {
-            e.printStackTrace();
+            LOGGER.error("Error while importing mail", e);
```

Comment:
Hum I don't understand the rational of what you are doing here.
I would just not catch the error. Instead of catching it loosing her stack-trace and rethrow an exception of the same type without any adding value.